package.json

```diff
@@ -23,30 +23,30 @@
   "author": "KissKissBankBank & Co",
   "homepage": "https://github.com/KissKissBankBank/kitten#readme",
   "dependencies": {
-    "@dnd-kit/core": "^4.0.1",
-    "@dnd-kit/modifiers": "^4.0.0",
-    "@dnd-kit/sortable": "^5.0.0",
-    "classnames": "^2.2.6",
+    "@dnd-kit/core": "^5.0.1",
+    "@dnd-kit/modifiers": "^5.0.0",
+    "@dnd-kit/sortable": "^6.0.0",
```

Comment:
Ça ne casse rien chez nous ?

Reply:
non, c'est le drag and drop utilisé uniquement dans le composant de réorganisation des rewards (dont le projet a été remis à… plus tard…)